Commands/deadchat.py

```
@@ -0,0 +1,130 @@
+        state = data["state"]
+
+        prefix = f"Messages in the past {minutes} minutes: {count}"
+
+        if state == "alive":
+            msg = f"{prefix} Alive chat Pog"
+        else:
+            msg = f"{prefix} deadchatxd"
+
+        self.send_privmsg(cmd.channel, msg)
+
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)